### DISCUSSION TITLE
Can't connect my Freenom .tk domain

### DISCUSSION THREAD

**Discussion by lone-architect** [community/community#23928] [Other Feature Feedback, Questions, & Ideas]
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/9/4/944484e701744af923eff06d8c8707993da77567.png" data-download-href="/uploads/short-url/l9DsY1w2cReU0ZwTu8CfS8laOj5.png?dl=1" title="11 - 11 54 24" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/50714470/181080064-dfb184e0-d58b-4862-8cb0-ad2c532c27e2.png" alt="11 - 11 54 24" data-base62-sha1="l9DsY1w2cReU0ZwTu8CfS8laOj5" width="690" height="385" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/4/944484e701744af923eff06d8c8707993da77567_2_690x385.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/4/944484e701744af923eff06d8c8707993da77567_2_1035x577.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/9/4/944484e701744af923eff06d8c8707993da77567_2_1380x770.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/4/944484e701744af923eff06d8c8707993da77567_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">11 - 11 54 24</span><span class="informations">2040×1139 106 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
I tried to register free domain, because AdMob not see app-ads.txt, when I use default GitHub domain. And after configuring DNS and GitHub Pages I still can’t use my own domain. What I should do?
<p>P.S. I’m really sorry for that terrible collage, but I couldn’t add more than 1 screenshot )</p>

**Comment by yoannchaudet** [comment]
<p>Hello,</p>
<p>If you just setup <code>jumperino.tk</code> it may not have propagated everywhere yet.</p>
<p>Just like Pages, I am unable to resolve it locally (or using the freenom’s DNS servers):</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/e/4eb96648b9bb5441bf797c54ba8bbd53aa250245.png" data-download-href="/uploads/short-url/beqstUcOpgRAtBFTt8plo2X0qwJ.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/14911070/181080066-d1db53ef-8659-4b82-8519-5f0eccf9f090.png" alt="image" data-base62-sha1="beqstUcOpgRAtBFTt8plo2X0qwJ" width="632" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/e/4eb96648b9bb5441bf797c54ba8bbd53aa250245_2_632x500.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/e/4eb96648b9bb5441bf797c54ba8bbd53aa250245_2_948x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/4/e/4eb96648b9bb5441bf797c54ba8bbd53aa250245_2_1264x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/e/4eb96648b9bb5441bf797c54ba8bbd53aa250245_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1280×1012 103 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by lone-architect** [comment]
<p>I did this about 10 hours ago. So I should just… Wait? )</p>
<p>UPD: Yeah, I just should wait, now all works fine</p>
